aten/src/ATen/native/Sorting.cpp

```diff
@@ -413,8 +435,9 @@ Tensor& quantile_out(
     const Tensor& self,
     const Tensor& q,
     optional<int64_t> _dim,
+    const std::string interpolation,
```

Comment:
This will break backwards compatibility for the C++ API. @mruberry Is there a better solution to add this parameter than adding 8 more overloads to native_functions?

Reply:
Good question. Probably not, unfortunately. Just mark the PR as "bc-breaking" with a label and add a "BC-breaking summary" at the top of the PR summary describing what's changed and why it's BC-breaking.

Reply:
I updated the PR summary with bc-breaking notes for adding interpolation parameter to torch.quantile function, please kindly confirm it and may update the content.